### PR TITLE
Release Google.Cloud.VMMigration.V1 version 1.0.0-beta01

### PR DESCRIPTION
**Pull request by jskeet**
Changes in this release:

Initial release.

### CODE REVIEW

File: apis/apis.json [hunk 1]
```diff
@@ -2996,6 +2996,24 @@
         "Translation"
       ]
     },
+    {
+      "id": "Google.Cloud.VMMigration.V1",
```
Comment (amanda-tarafa) [inline apis/apis.json:3000]: Should this one go after the Google.Cloud.Video* ones?

**Review by amanda-tarafa — COMMENTED** [review]
Also I think the first and second commit are named the same.

File: apis/apis.json (hunk 1 above)
Comment (jskeet) [reply apis/apis.json:3000]: Nope - the capital "M" means that under ordinal sorting, it goes before it. (This insertion was autogenerated, so should be okay.)

**Comment by jskeet** [comment]
Will fix the second commit title - thanks for finding.

**Review by amanda-tarafa — APPROVED** [review]
LGTM

**Pull request merged**